package/MDAnalysis/coordinates/DCD.py

```diff
@@ -188,13 +188,10 @@ def _read_next_timestep(self, ts=None):
         if self._frame == self.n_frames - 1:
             raise IOError('trying to go over trajectory limit')
         if ts is None:
-            # use a copy to avoid that ts always points to the same reference
-            # removing this breaks lammps reader
-            ts = self.ts.copy()
+            ts = self.ts
```

Comment:
My understanding was that the majority of devs here did _not_ want to change the behavior of the ts being a copy but instead deprecate the current behavior and change it in 3.0, or did I misunderstand the discussion??

Reply:
Ah sorry I had misinterpreted and thought this whole PR was shifted to 3.0, but what you propose makes a lot more sense.  Deprecate and change for 3.0.

Reply:
You, know, doing it for 3.0 would be a possibility. But I suggest we do most of the changes right now _except removing the copy_ for two reasons
1. We need to put in a deprecation note (docs and CHANGELOG). Better sooner than later.
2. If time-to-close-PR is a metric for CZI & friends then we rather close/merge PRs sooner ;-)

I don't know if we should issue a DeprecationWarning — do people have opinions? If yes, the I think it should be on creating a DCDReader _or_ in `Timestep.__init__` _if_ we only rarely create new Timestep instances. If we generate a new ts often as part of DCD reading (I can't remember...) then better only once in the DCDReader init. Open for ideas...

Reply:
I have put it in DCDReader `init`